packages/rpc/src/chainStorageWatcher.ts

```diff
@@ -1,9 +1,10 @@
 /* eslint-disable no-use-before-define */
 /* eslint-disable import/extensions */
-import type { FromCapData } from '@endo/marshal';
-import type { UpdateHandler } from './types';
+/* eslint-disable import/no-extraneous-dependencies */
+import { makeMarshal } from 'marshal';
```

Comment:
We should really avoid undeclared dependencies. I think this is just as simple as:
```suggestion
import { makeMarshal } from './marshal';
```

Reply:
Interesting... done